src/you_get/extractors/mgtv.py

```diff
@@ -75,8 +75,9 @@ def prepare(self, **kwargs):
                 size_this_stream = 0
                 stream_fileid_list = []
                 for i in segment_list_this:
-                    _, container_this_stream, size_this_seg = url_info(i)
-                    size_this_stream += size_this_seg
+                    if 'json_output' not in kwargs and not kwargs['json_output']:
```

Comment:
@wwqgtxx We are not complaining about this PR: we are talking about a potential bug deeply hiding within EVERY ```VideoExtractor``` . If that is the case, we'd better fix it for it could be critical. Otherwise, can you kindly explain why you are trying to do this hacking? What unexpected behaviour are you having? 

Many thanks.